packages/flutter_test/lib/src/controller.dart

```diff
@@ -17,8 +17,10 @@ import 'test_pointer.dart';
 ///
 /// For a variant of this class suited specifically for unit tests, see [WidgetTester].
 class WidgetController {
+  /// Creates a widget controller that uses the given binding.
   WidgetController(this.binding);
 
+  /// The binding instance used by the widget controller.
```

Comment:
there's only one binding instance, by definition. Maybe "A reference to the current instance of the binding" or something would make that clearer.